docs/classes/_builder_.builder.md

```diff
@@ -0,0 +1,451 @@
+[@xcoobee/payment-sdk](../README.md) › [Globals](../globals.md) › ["Builder"](../modules/_builder_.md) › [Builder](_builder_.builder.md)
+
+# Class: Builder
```

Comment:
Can we do something to indicate to the end-user that this class is not apart of the public API? Maybe annotating the class as private will do the trick.

I'd be perfectly fine if private code is not included in this documentation. Most doc generators I've used leave out private code by default. I'm not sure about `typedoc` -- never used it before.

Reply:
@dhurlburtusa @Bilal-S I`m not sure that we need to hide it or mark as private. My intent was to create generic api which could help users build custom urls. If users are not satisfied with public methods they can use `Builder` to generate custom link